config/initializers/base.rb

```diff
@@ -0,0 +1,16 @@
+module ActiveRecord
+  class Base
+    def self.reset_pk_sequence
+      case ActiveRecord::Base.connection.adapter_name
+      when 'SQLite'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.